src/test/regress/sql/failure_create_index_concurrently.sql

```diff
@@ -0,0 +1,65 @@
+--
```

Comment:
Considering this [comment](https://github.com/citusdata/citus/pull/2244#issuecomment-402990403) and thinking about this PR as the main one for bare DDLs, I could suggest three more tests:

- Test with replication factor 1 as well (I think the default in the regression test suite is two, but, please double check that)
- Add tests for DROP INDEX CONCURRENTLY as well.
- Since the `CREATE/DROP INDEX CONCURRENTLY` uses sequential execution mode, we could kill/cancel the second (or later) commands over a single connection. For example, if you create 8 shards, we'd end up with 4 shards per worker and 4 shards are modified over the same connection per node. So, say if you have `whatevercase().after(2).kill()`, it'd kill the connection after seeing the same command 2 times. An example can be found [here](https://github.com/citusdata/citus/pull/2212/files#diff-1490fb849241ff1456eb927ed102394aR1194)